tests/actions/EnforceActionExportRestrictions.ts

```diff
@@ -17,16 +17,77 @@ describe('ReportUtils', () => {
         expect(ReportUtils.getReport).toBeUndefined();
     });
 
-    it('does not export isOneTransactionReport', () => {
-        // @ts-expect-error the test is asserting that it's undefined, so the TS error is normal
-        expect(ReportUtils.isOneTransactionReport).toBeUndefined();
-    });
+    // TODO: Re-enable this test when isOneTransactionReport is fixed https://github.com/Expensify/App/issues/64333
+    // it('does not export isOneTransactionReport', () => {
+    //     // @ts-expect-error the test is asserting that it's undefined, so the TS error is normal
+    //     expect(ReportUtils.isOneTransactionReport).toBeUndefined();
+    // });
 
     it('does not export getPolicy', () => {
         // @ts-expect-error the test is asserting that it's undefined, so the TS error is normal
         expect(ReportUtils.getPolicy).toBeUndefined();
     });
 
+    it('does not export getIconsForChatThread', () => {
+        // @ts-expect-error the test is asserting that it's undefined, so the TS error is normal
+        expect(ReportUtils.getIconsForChatThread).toBeUndefined();
```

Comment:
I don't want anyone to accidentally hardcode avatars by exporting these functions